index.html

```diff
@@ -6,67 +6,60 @@
     <meta name="keywords" content="HTML, CSS, JavaScript">
     <meta name="author" content="Tiffany Cooper">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Skills Section</title>
 </head>
 <body> 
 <button class="tablink" onclick="openPage('About', this, 'red')">About</button>
```

Comment:
These aren't functional. Maybe they are copied from an example that defined an `openPage` function? I'd suggest using plain `<a>` tags.